easybuild/easyconfigs/g/GPAW/GPAW-1.3.0-intel-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = "PythonPackage"
+
+name = 'GPAW'
+version = '1.3.0'
+
+homepage = 'https://wiki.fysik.dtu.dk/gpaw/'
+description = """GPAW is a density-functional theory (DFT) Python code based on the projector-augmented wave (PAW)
+ method and the atomic simulation environment (ASE). It uses real-space uniform grids and multigrid methods or
+ atom-centered basis-functions."""
+
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+source_urls = ['https://pypi.python.org/packages/11/59/bfb6102911f9c0c35bfd0c1f3f51e0f4c31523b9f700674d18e4f60d7081/']
+sources = [SOURCELOWER_TAR_GZ]
```

Comment:
Hi @Micket - welcome to `EasyBuild`, and thanks for your contribution.

I'll just add a couple of comments about this PR, to meet the guidelines.

For example, we now ask people to include a checksum for the sources. You can add this automatically with the `--inject-checksums` command-line option. Note that we use `sha256`.